sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -590,9 +590,11 @@ def get_coder(self, coder_id):
     if coder_proto.spec.spec.urn:
       return self.context.coders.get_by_id(coder_id)
     else:
+      payload = coder_proto.spec.spec.payload
+      if isinstance(payload, bytes):
```

Comment:
Do we need the `if`? Is it not always `bytes`?

Reply:
I'm not a hundred per cent sure. @robertwb Do we need check here?